packages/contracts/src/factory/VaultFactory.sol

```diff
@@ -8,7 +8,7 @@ import { AccessControl } from "@openzeppelin/contracts/access/AccessControl.sol"
 /**
  * @notice - A factory contract to create vault contract
  */
-abstract contract CredbullVaultFactory is AccessControl {
+abstract contract VaultFactory is AccessControl {
```

Comment:
looks good to me.  recommend we decide on a few platforms/protocols we can use for inspiration / style, e.g.

1. OpenZeppelin - de-facto standard for many ERC implementations https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts
2. Centrifuge - we need to understand their solution anyhow as our fund: https://github.com/centrifuge/liquidity-pools/tree/main/src